src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/ConfigCommands/ConfigRunners.cs

```diff
@@ -195,7 +197,7 @@ public static void LogSections(IEnumerable<string> sections, Settings settings,
         /// <param name="logger"></param>
         public static void LogSectionItems(IEnumerable<SettingItem> items, ILogger logger)
         {
-            foreach (SettingItem item in items)
+            foreach (SettingItem item in items.Reverse())
```

Comment:
Doesn't this reverse the change above?

Reply:
This reverses the order of the individual items in the file path groups, not the order of the groups themselves. Now that you mentioned it, it doesn't make sense to do this operation when we're already reversing the order of the groups. I'll change that!

Reply:
Can we get a test as well?